### ISSUE TITLE
Add log dirs alter operation

### ISSUE THREAD

**Issue by germanosin** [provectus/kafka-ui#708]
Add endpoint: PATCH /api/clusters/{clusterName}/brokers/{id}/logDirs
```json
[
    {
        "topic": "topicName",
        "partition": 0,
        "logDir": "/var/data/kafka"
    }, ...
]
```

Use AdminClient alterReplicaLogDirs operation for it

**Comment by workshur** [comment]
It looks like UI team needs an endpoint to fetch the list of all available logDirs

**Comment by germanosin** [comment]
@workshur you can get them here: https://github.com/provectus/kafka-ui/issues/707

**Comment by tdavletov** [comment]
Implemented here - #723
